core/qt-init.cpp

```diff
@@ -107,10 +97,40 @@ void init_qt_late()
 			}
 		}
 	}
+	// when creating our language names, we didn't define generic languages with additional
+	// country specific extensions, instead we included the "primary" country in the
+	// translation designations. This breaks the way Qt finds fall-back translations.
+	// Changing this now seems rather tedious, so instead we manually create a few
+	// obvious fallbacks
+	QPair<QLocale::Language, QLocale::Country> parents[] = {
+		{ QLocale::German, QLocale::Germany },
+		{ QLocale::Portuguese, QLocale::Brazil },
```

Comment:
Shouldn't this be `{ QLocale::Portuguese, QLocale::Portugal },` since the pt_PT translation is complete, and pt_BR is not?

Reply:
Oops. Yes. Of course. What was I thinking... likely that we initially had a better Brazilian Portuguese translation, but that's long long past...